cocotb/binary.py

```diff
@@ -303,12 +303,35 @@ def is_resolvable(self):
         """Does the value contain any ``X``'s?  Inquiring minds want to know."""
         return not any(char in self._str for char in BinaryValue._resolve_to_error)
 
-    value = property(get_value, set_value, None,
-                     "Integer access to the value. **deprecated**")
-    integer = property(get_value, set_value, None,
-                       "The integer representation of the underlying vector.")
-    signed_integer = property(get_value_signed, set_value, None,
-                              "The signed integer representation of the underlying vector.")
+    @property
+    def value(self):
+        "Integer access to the value. **deprecated**"
```

Comment:
I think we should take this opportunity to deprecate this interface by adding a warning and removing the message from the docstring.

Reply:
Tracked by #1503, marking as resolved here.